gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
This is not good: you replaced all the symlinks by regular files. Can you put the symlinks back? This PR shouldn't touch the `make_wiki` functionality at all IMO.